src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -71,6 +72,7 @@ const bundleJsFile = async function ({
       processImports: config.processDynamicNodeImports !== false,
       srcDir,
     }),
+    getNodeBuiltinPlugin(),
```

Comment:
Might be a micro-optimisation, but I think we'd benefit from running this plugin first, since marking a path as external will avoid running the other two (more expensive) plugins.